CHANGELOG.md

```diff
@@ -7,7 +7,13 @@ All notable changes to this project will be documented in this file.
 
 #### API breaking changes
 
-TBD
+- Removing `MaskDesignable` public methods:
+   - `maskCircle()`, use instead: `view.maskType = String(MaskType.Circle)`
```

Comment:
You are right @lexrus, no reason to do `String()` instead of `rawValue`. Thanks for the change @JakeLin!